Tests/StencilSwiftKitTests/CallNodeTests.swift

```diff
@@ -11,64 +11,64 @@ import XCTest
 class CallNodeTests: XCTestCase {
   func testParser() {
     let tokens: [Token] = [
-      .block(value: "call myFunc"),
+      .block(value: "call myFunc")
     ]
-    
+
     let parser = TokenParser(tokens: tokens, environment: stencilSwiftEnvironment())
     guard let nodes = try? parser.parse(),
       let node = nodes.first as? CallNode else {
       XCTFail("Unable to parse tokens")
       return
     }
-    
+
     XCTAssertEqual(node.variableName, "myFunc")
     XCTAssertEqual(node.arguments, [])
   }
-  
+
   func testParserWithArguments() {
     let tokens: [Token] = [
-      .block(value: "call myFunc a b c"),
+      .block(value: "call myFunc a b c")
     ]
-    
+
     let parser = TokenParser(tokens: tokens, environment: stencilSwiftEnvironment())
     guard let nodes = try? parser.parse(),
       let node = nodes.first as? CallNode else {
       XCTFail("Unable to parse tokens")
       return
     }
-    
+
     XCTAssertEqual(node.variableName, "myFunc")
     XCTAssertEqual(node.arguments, [Variable("a"), Variable("b"), Variable("c")])
   }
-  
+
   func testParserFail() {
     do {
       let tokens: [Token] = [
         .block(value: "call")
       ]
-      
+
       let parser = TokenParser(tokens: tokens, environment: stencilSwiftEnvironment())
       XCTAssertThrowsError(try parser.parse())
     }
   }
-  
-  func testRender() {
+
+  func testRender() throws {
```

Comment:
I didn't even know XCTest would still recognize functions as tests when they are marked with `throws`, I thought only `func test<Whatever>()` functions with this exact signature type would be recognized! (wait, that test actually _does_ get executed, right?)